### PR TITLE
Ocd 2695 - Standardize de-selection of a value for all multi-select fields when editing a listing

### PR DESCRIPTION
**Pull request by tmy1313**

### CODE REVIEW

File: src/app/admin/components/certifiedProduct/management.html [hunk 1]
```diff
@@ -372,6 +372,9 @@ <h3>Manage Surveillance Activity</h3>
         </div>
         <div class="col-sm-12" ng-if="vm.workType === 'merge' && vm.mergeType === 'developer'">
           <label for="developer-merge-select" class="control-label admin-label-spaced">Select developers to merge</label>
+          <div class="pull-right" ng-if="vm.mergingDevelopers && vm.mergingDevelopers.length !== 0">
```
Comment (andlar) [inline src/app/admin/components/certifiedProduct/management.html]: All of these look the same, so I'm going to put my comment in one place, but they're applicable to them all

Question(s)
* Does this need to be in a div of its own?
* Could it just have the `ng-if` and `pull-right' classes applied to the button itself?

Requested change(s):
* Please add an ID, unique to the thing that's being cleared, so our AQA team can target these buttons
* Please add text in the "Clear Selection(s)" thing, probably hidden in a `<span class="sr-only">` that names the selection that's being cleared, for the purposes of 508 compliance
* Oh, and I think all `<button>`s need to have the class `btn` in addition to whatever other classes are needed

File: src/app/components/listing_details/criteriaModal.html [hunk 2]
```diff
@@ -33,15 +33,24 @@ <h4 class="modal-title" id="modalLabel">Certification Criteria: {{ vm.cert.numbe
...
             <td><label class="control-label" for="data_g1MacraMeasures{{ vm.cert.number }}">Measure<span ng-if="vm.cert.g1MacraMeasures.length !== 1">s</span> Successfully Tested for G1</label></td>
             <td>
-              <select multiple data-ng-attr-size="{{ vm.cert.allowedMacraMeasures.length + 1 }}" class="form-control input-sm" id="data_g1MacraMeasures{{ vm.cert.number }}" name="data_g1MacraMeasures" ng_model="vm.cert.g1MacraMeasures" ng-options="m as m.abbreviation for m in vm.cert.allowedMacraMeasures | orderBy: 'abbreviation' track by m.id"></select></td>
+              <div class="pull-right" ng-if="vm.cert.g1MacraMeasures && vm.cert.g1MacraMeasures.length !== 0">
+                <button class="btn-small btn-link" ng-click="vm.cert.g1MacraMeasures=[];">Clear Selection(s)</button>
+              </div>
+              <select multiple data-ng-attr-size="{{ vm.cert.allowedMacraMeasures.length + 1 }}" class="form-control input-sm" id="data_g1MacraMeasures{{ vm.cert.number }}" name="data_g1MacraMeasures" ng_model="vm.cert.g1MacraMeasures" ng-options="m as m.abbreviation for m in vm.cert.allowedMacraMeasures | orderBy: 'abbreviation' track by m.id"></select>
```
Comment (andlar) [inline src/app/components/listing_details/criteriaModal.html]: For these macra measure ones, did you change anything with them? I can't see a difference, but git is showing one

Comment (tmy1313) [reply src/app/components/listing_details/criteriaModal.html]: I did not intentionally make any changes to the macra measures.  Not sure why they are showing up as changed.

File: src/app/admin/components/certifiedProduct/management.html [hunk 3]
```diff
@@ -372,6 +372,7 @@ <h3>Manage Surveillance Activity</h3>
         </div>
         <div class="col-sm-12" ng-if="vm.workType === 'merge' && vm.mergeType === 'developer'">
           <label for="developer-merge-select" class="control-label admin-label-spaced">Select developers to merge</label>
+          <button id="clear-selection-developers" class="btn btn-small btn-link pull-right" ng-click="vm.mergingDevelopers=[];" ng-if="vm.mergingDevelopers && vm.mergingDevelopers.length !== 0">Clear Selection(s)<span class="sr-only">Developers</span></button>
```
Comment (andlar) [inline src/app/admin/components/certifiedProduct/management.html]:
```suggestion
          <button id="clear-selection-developers" class="btn btn-small btn-link pull-right" ng-click="vm.mergingDevelopers=[];" ng-if="vm.mergingDevelopers && vm.mergingDevelopers.length !== 0">Clear Selection(s)<span class="sr-only"> of Developers</span></button>
```

File: src/app/admin/components/certifiedProduct/management.html (hunk 1 above)
Comment (andlar) [reply src/app/admin/components/certifiedProduct/management.html]: Good changes. Added a space and "of" so something reading the text would read it as a sentence

File: src/app/components/listing_details/criteriaModal.html (hunk 2 above)
Comment (andlar) [reply src/app/components/listing_details/criteriaModal.html]: Fine by me. Was curious if I couldn't see something

**Pull request merged**